app/experimenter/experiments/models.py

```diff
@@ -660,6 +665,64 @@ def firefox_channel_sort():
             output_field=models.IntegerField(),
         )
 
+    def clone(self, name, user):
```

Comment:
This method is a work of art 

![](http://fatcatart.ru/wp-content/uploads/2012/01/Leonardo_Mona_Lisa_cat_sm.jpg)